pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -207,14 +207,14 @@ func init() {
 		Rules: []rbac.PolicyRule{
 			rbac.NewRule("get", "create", "update").Groups(networkGroup, legacyNetworkGroup).Resources("clusternetworks").RuleOrDie(),
 			rbac.NewRule("get", "list").Groups(networkGroup, legacyNetworkGroup).Resources("egressnetworkpolicies").RuleOrDie(),
-			rbac.NewRule("get", "list", "create", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("hostsubnets").RuleOrDie(),
-			rbac.NewRule("get", "list", "create", "update", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("netnamespaces").RuleOrDie(),
+			rbac.NewRule("get", "list", "watch", "create", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("hostsubnets").RuleOrDie(),
+			rbac.NewRule("get", "list", "watch", "create", "update", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("netnamespaces").RuleOrDie(),
 			rbac.NewRule("get", "list").Groups(kapiGroup).Resources("pods").RuleOrDie(),
 			rbac.NewRule("list").Groups(kapiGroup).Resources("services").RuleOrDie(),
 			rbac.NewRule("list").Groups(kapiGroup).Resources("namespaces").RuleOrDie(),
 			rbac.NewRule("get").Groups(kapiGroup).Resources("nodes").RuleOrDie(),
```

Comment:
The master is using a shared informer, I'll add the permissions explicitly though.